### PR TITLE
#14 Add daily menu to Lokál Hamburk

### PR DESCRIPTION
**Pull request by johnsoncze**

### CODE REVIEW

File: lib/service/lokal-hamburk.js [hunk 1]
```diff
@@ -23,19 +23,22 @@ module.exports = function () {
...
-
-        items.push({
+        var item = {
           item: getItem(lunchMenu[i]),
           price: lunchMenu[i].price,
           amount: getAmount(lunchMenu[i])
-        })
+        }
+        if(item.item.match(/^Expres/))
+          items.unshift(item) //Expres first
```
Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Expected '{' and instead saw 'items'.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l39)

File: lib/service/lokal-hamburk.js [hunk 2]
```diff
@@ -23,19 +23,22 @@ module.exports = function () {
...
+        else
+          items.push(item)
```
Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Expected '{' and instead saw 'items'.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l41)

File: lib/service/lokal-hamburk.js [hunk 3]
```diff
@@ -23,19 +23,22 @@ module.exports = function () {
...
         }
-
-        items.push({
+        var item = {
           item: getItem(lunchMenu[i]),
           price: lunchMenu[i].price,
           amount: getAmount(lunchMenu[i])
-        })
+        }
+        if(item.item.match(/^Expres/))
```
Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Expected { after 'if' condition.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l38)

Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Keyword "if" must be followed by whitespace.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l38)

File: lib/service/lokal-hamburk.js (hunk 1 above)
Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Expected space or tab after "//" in comment.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l39)

File: lib/service/lokal-hamburk.js [hunk 4]
```diff
@@ -23,19 +23,22 @@ module.exports = function () {
   var processMenu = function (obj, options, next) {
```
Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l23)

File: lib/service/lokal-hamburk.js [hunk 5]
```diff
@@ -23,19 +23,22 @@ module.exports = function () {
...
-        items.push({
+        var item = {
           item: getItem(lunchMenu[i]),
           price: lunchMenu[i].price,
           amount: getAmount(lunchMenu[i])
-        })
+        }
+        if(item.item.match(/^Expres/))
+          items.unshift(item) //Expres first
+        else
```
Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Expected { after 'else'.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l40)

**Comment by Gobie** [comment]
Nepouštíš `npm test` před pushem co? :)

**Comment by johnsoncze** [comment]
Ocividne ne no :) ale zil jsem v tom ze jo. Skoro bych rekl, ze jsem pred
pushem udelal posledni upravu a pak uz jsem test nepustil :)

pá 8. 4. 2016 v 8:42 odesílatel Michal Brašna <notifications@github.com>
napsal:

File: lib/service/lokal-hamburk.js [hunk 6]
```diff
@@ -23,19 +23,23 @@ module.exports = function () {
   var processMenu = function (obj, options, next) {
```
Comment (Gobie) [inline lib/service/lokal-hamburk.js]: Codacy Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/Socialbakers/luncher/file/2376802332/issues/source?bid=3218200&fileBranchId=3224190#l23)

**Pull request merged**